drivers/timer/Kconfig

```diff
@@ -154,7 +154,6 @@ config NRF_RTC_TIMER
 	default y
 	depends on CLOCK_CONTROL_NRF
 	select TICKLESS_CAPABLE
-	select COUNTER_RTC1
```

Comment:
How did this come back?!  This was removed in #12475 by 85d10e02beb69219cb5b8391e990b05d54e783c0.

It came back because a rebase later added 5ef145da60a8d611deaa73e140cc9a2dbbf9b198 by repeating fragments from a really old commit.

Rewrites of the base branch **must** be carefully monitored to confirm they do not discard material from merged commits or we'll have no idea what we've got in the implementation.